docs/docs/core/00-baseapp.md

```diff
@@ -131,34 +131,55 @@ Naturally, developers can add additional `options` based on their application's
 ## State Updates
 
 The `BaseApp` maintains two primary volatile states and a root or main state. The main state
```

Comment:
I think, if I recall 4, is checkState, preparePropossal, processPropossal and Deliver, but I may be wrong. 
```suggestion
The `BaseApp` maintains four primary volatile states and a root or main state. The main state
```

Reply:
Yes thanks, typo there